### PR TITLE
Microsoft Azure blob storage integration

### PR DESCRIPTION
**Pull request by tcnksm**
This PR  allows to use to Microsoft Azure blob storage service (https://azure.microsoft.com/en-us/documentation/articles/storage-introduction/#blob-storage) for backend.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.pinterest</groupId>
     <artifactId>secor</artifactId>
-    <version>0.16-SNAPSHOT</version>
+    <version>0.17-SNAPSHOT</version>
```
Comment (HenryCaiHaiying) [inline pom.xml]: No need to bump up the version.  Bump the version would require us to do a maven publish, and that's a manual process for us.

**Comment by HenryCaiHaiying** [comment]
Looks fine, small comment on version in pom.xml

**Comment by tcnksm** [comment]
Fixed `pom.xml`

**Comment by HenryCaiHaiying** [comment]
Looks good, thanks for the contribution

**Comment by tcnksm** [comment]
Thanks

**Comment by optiomas** [comment]
This hasn't been documented in the README.

**Comment by HenryCaiHaiying** [comment]
Please feel free to submit a PR to add this in README

**Pull request merged**